tests/message/test_messages.py

```diff
@@ -51,6 +52,8 @@ def test_message_want_to_compute_task(self):
         max_memory_size = random.randint(1, 2**10)
         num_cores = random.randint(1, 2**5)
         concent_enabled = random.random() > 0.5
+        provider_public_key = cryptography.ECCx(None).raw_pubkey
```

Comment:
hmm.... why this particular value? the message normally contains:

```
encode_key_id(cryptography.ECCx(None).raw_pubkey))
```

so, a hex-encoded representation of the public key... 

so, if we want to stay true to the actually expected content of those fields, we should use the above for both `provider_public_key` and `provider_ethereum_public_key` ... and - as is the case here - if we don't care about the value being a representation of an actual public key, then we can use a random string for both ...

Reply:
@shadeofblue :+1: done, I think it is better to use real values when possible.